registry/config.go

```diff
@@ -32,6 +32,7 @@ const (
 	REGISTRYSERVER = DEFAULT_V2_REGISTRY
 	INDEXSERVER    = DEFAULT_V1_REGISTRY + "/v1/"
 	INDEXNAME      = "docker.io"
+	NOTARYSERVER   = "https://notary.docker.io"
 
 	// INDEXSERVER = "https://registry-stage.hub.docker.com/v1/"
```

Comment:
Ya but since you are here...

> On Jul 21, 2015, at 23:20, David Lawrence <notifications@github.com> wrote:
> 
> In registry/config.go:
> 
> > @@ -32,6 +32,7 @@ const (
> >  	REGISTRYSERVER = DEFAULT_V2_REGISTRY
> >  	INDEXSERVER    = DEFAULT_V1_REGISTRY + "/v1/"
> >  	INDEXNAME      = "docker.io"
> > +	NOTARYSERVER   = "https://notary.docker.io"
> >  
> >  	// INDEXSERVER = "https://registry-stage.hub.docker.com/v1/"
> That's in master, you don't need to ask us for permissions to remove it :-)
> 
> —
> Reply to this email directly or view it on GitHub.
>